corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -111,3 +139,40 @@ def get_domains(self):
     def migrate_app(self, app):
         """Return the app dict if the doc is to be saved else None"""
         raise NotImplementedError()
+
+    def set_filenames(self):
+        self.domain_list_filename = __class__.__name__ + "app_migration_command_domain_list.txt"
+        self.domain_progress_number_filenane = __class__.__name__ + "app_migration_command_domain_progress.txt"
```

Comment:
must use `self.__class__` otherwise it will always be the name of the base class:
```suggestion
        self.domain_list_filename = self.__class__.__name__ + "app_migration_command_domain_list.txt"
        self.domain_progress_number_filenane = self.__class__.__name__ + "app_migration_command_domain_progress.txt"
```

Reply:
Oh right, because the class names are just `Command`. No need to change it again but you could try `self.__module__` which should be the filename of the file where the class is defined.